modules/kafka/kafka.go

```diff
@@ -165,20 +217,61 @@ func WithClusterID(clusterID string) testcontainers.CustomizeRequestOption {
 
 // Brokers retrieves the broker connection strings from Kafka with only one entry,
 // defined by the exposed public port.
+//
+// Example Output: localhost:<random_port>
 func (kc *KafkaContainer) Brokers(ctx context.Context) ([]string, error) {
 	host, err := kc.Host(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	port, err := kc.MappedPort(ctx, publicPort)
+	port, err := kc.MappedPort(ctx, PublicLocalhostPort)
 	if err != nil {
 		return nil, err
 	}
 
 	return []string{fmt.Sprintf("%s:%d", host, port.Int())}, nil
 }
 
+// BrokersByHostDockerInternal retrieves broker connection string suitable when
+// running 2 containers in the default docker network
+//
+// Example Output: host.docker.internal:<random_port>
+func (kc *KafkaContainer) BrokersByHostDockerInternal(ctx context.Context) ([]string, error) {
```

Comment:
question: why is this returning a slice when its hard coded to a single entry? More below

Reply:
Was just following the previous API for `Brokers()`. I suppose there is a future where maybe we do setup a multinode cluster and having the same API to return the list of brokers might be a good idea so the public API doesn't change if/when that happens.